geomesa-ui/src/main/ui/bower.json

```diff
@@ -14,6 +14,7 @@
     "lodash": "~2.4.1",
     "moment": "~2.7.0",
     "moment-timezone": "~0.1.0",
-    "openlayers" : "https://github.com/openlayers/ol3/releases/download/v3.2.1/v3.2.1.zip"
+    "openlayers" : "https://github.com/openlayers/ol3/releases/download/v3.2.1/v3.2.1.zip",
+    "openlayers2": "https://github.com/openlayers/openlayers/releases/download/release-2.13.1/OpenLayers-2.13.1.zip"
```

Comment:
Have you looked into https://github.com/52North/wps-js at all? I haven't yet, but seems like it might be viable and would remove the need for OL2 (is that true or are you using it for something else as well?)